mysql/datadog_checks/mysql/data/conf.yaml.example

```diff
@@ -251,6 +251,96 @@ instances:
     #   - <TOP_K_LIMIT>
     #   - <BOTTOM_K_LIMIT>
 
+    ## Configure collection of statement samples
+    #
+    statement_samples:
+
+        ## @param enabled - boolean - optional - default: false
+        ## Enable collection of statement samples. Requires `deep_database_monitoring: true`.
```

Comment:
Is this content generated from the spec file in this same PR?

Reply:
Yes, just updated https://github.com/DataDog/integrations-core/pull/8629/commits/d7cfa8c7123fbdc00f2b7b3329828479599d2381